dev/benchmarks/macrobenchmarks/lib/src/web/bench_platform_view_infinite_scroll.dart

```diff
@@ -3,30 +3,32 @@
 // found in the LICENSE file.
 
 import 'dart:async';
-import 'dart:html' as html;
+import 'dart:js_interop';
 
 import 'package:flutter/material.dart';
+import 'package:web/web.dart' as web;
 
 // TODO(mdebbar): flutter/flutter#55000 Remove this conditional import once
 // web-only dart:ui_web APIs are exposed from a dedicated place.
 import 'platform_views/non_web.dart'
-    if (dart.library.html) 'platform_views/web.dart';
+    if (dart.library.js_interop) 'platform_views/web.dart';
 import 'recorder.dart';
 
 const String benchmarkViewType = 'benchmark_element';
 
 void _registerFactory() {
   platformViewRegistry.registerViewFactory(benchmarkViewType, (int viewId) {
-    final html.Element htmlElement = html.DivElement();
-    htmlElement.id = '${benchmarkViewType}_$viewId';
-    htmlElement.innerText = 'Google';
+    final web.HTMLElement htmlElement = web.document.createElement('div'.toJS)
+        as web.HTMLDivElement;
+    htmlElement.id = '${benchmarkViewType}_$viewId'.toJS;
+    htmlElement.innerText = 'Google'.toJS;
     htmlElement.style
-      ..width = '100%'
-      ..height = '100%'
-      ..color = 'black'
-      ..backgroundColor = 'rgba(0, 255, 0, .5)'
-      ..textAlign = 'center'
-      ..border = '1px solid black';
+      ..setProperty('width'.toJS, '100%'.toJS)
```

Comment:
btw - I run into these on my migration too, I think we are going to hit these very often and we can do better in `package:web` for this.

Looking more closely, dart:html has a list of properties for CSS that we use to generate the full set of accessors (see https://github.com/dart-lang/sdk/blob/main/tools/dom/scripts/CSSPropertyNames.in)

Filed an issue: https://github.com/dart-lang/web/issues/47